### PR TITLE
Use purescript-simple-dom

### LINKED ISSUE #67: Use simple-dom appendChild
I just opened a pull request: https://github.com/aktowns/purescript-simple-dom/pull/26

Things like the ToDo app will no longer need a foreign import! :fireworks:

### PR DESCRIPTION
**Pull request by puffnfresh**
The API now exposes simple-dom's HTMLElement and the examples now use
appendToBody and setInterval.

Fixes #67

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -32,6 +32,6 @@
     "purescript-lists": "~0.6.0"
   },
   "devDependencies": {
-    "purescript-ace": "~0.2.0"
+    "purescript-ace": "a4ee2f1a00943b5e3d8098642ae7c3a1517b6bd4"
```
Comment (paf31) [inline bower.json]: We should probably make a major version bump on `purescript-ace`, right?

**Comment by paf31** [comment]
:+1: Looks nice.

File: bower.json (hunk 1 above)
Comment (puffnfresh) [inline bower.json]: Yep, I'll update this PR when that's done.

**Comment by paf31** [comment]
Ack, sorry, I thought this would be fine, but I just broke your PR. It should be a simple merge though.

**Pull request merged**